assets/js/modules/analytics/components/dashboard/AnalyticsAdSenseDashboardWidgetLayout.js

```diff
@@ -25,18 +25,21 @@ import { __, _n, sprintf } from '@wordpress/i18n';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { MODULES_ADSENSE, DATE_RANGE_OFFSET } from '../../../adsense/datastore/constants';
+import { DATE_RANGE_OFFSET } from '../../../adsense/datastore/constants';
+import { MODULES_ANALYTICS } from '../../../analytics/datastore/constants';
 import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
 import Layout from '../../../../components/layout/Layout';
 import { getCurrentDateRangeDayCount } from '../../../../util/date-range';
-import { generateDateRangeArgs } from '../../../adsense/util/report-date-range-args';
 const { useSelect } = Data;
 
 const AnalyticsAdSenseDashboardWidgetLayout = ( { children } ) => {
-	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+	const { startDate, endDate } = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
 		offsetDays: DATE_RANGE_OFFSET,
 	} ) );
-	const accountSiteURL = useSelect( ( select ) => select( MODULES_ADSENSE ).getServiceReportURL( generateDateRangeArgs( dateRangeDates ) ) );
+	const accountSiteURL = useSelect( ( select ) => select( MODULES_ANALYTICS ).getServiceReportURL(
```

Comment:
"accountSiteURL" is more of an AdSense terminology (the store also has a `getServiceAccountSiteURL` selector). Let's call this `analyticsReportURL` instead since the selector is using Analytics rather than AdSense now.